v21.1/backup.md

```diff
@@ -30,6 +30,7 @@ Because CockroachDB is designed with high fault tolerance, these backups are des
 - `BACKUP` is a blocking statement. To run a backup job asynchronously, use the `DETACHED` option. See the [options](#options) below.
 - Backups will export [Enterprise license keys](enterprise-licensing.html) during a [full cluster backup](#backup-a-cluster). When you [restore](restore.html) a full cluster with an Enterprise license, it will restore the Enterprise license of the cluster you are restoring from.
 - [Interleaving data](interleave-in-parent.html) is disabled in v21.1 by default, and will be permanently removed from CockroachDB in a future release. CockroachDB versions v21.2 and later will not be able to read or restore backups that include interleaved data. To backup interleaved data in v21.1, a `BACKUP` statement must include the [`INCLUDE_DEPRECATED_INTERLEAVES` option](#include-deprecated-interleaves).
+- [Zone configurations](configure-zone.html) present on the destination cluster prior to a restore will be **overwritten** during a [cluster restore](restore.html#full-cluster) with the zone configurations from the [backed up cluster](#backup-a-cluster). If there were no customized zone configurations on the cluster when the backup was taken, then the restore will inherit zone configurations from the [`RANGE DEFAULT` configuration](configure-replication-zones.html#view-the-default-replication-zone).
```

Comment:
> If there were no customized zone configurations on the cluster when the backup was taken, then the restore will inherit zone configurations from the... 

Could we rephrase "The restore will inherit" to clarify what the user can expect to see in their destination cluster? e.g. .. "if... then, after the restore, the destination cluster will use the Zone config from the range default"